spec/factories/patients.rb

```diff
@@ -108,6 +108,20 @@
       end
     end
 
+    trait :consent_refused_with_notes do
+      after(:create) do |patient, evaluator|
+        create(
+          :consent,
+          :refused,
+          :from_mum,
+          campaign: evaluator.campaign,
+          reason_for_refusal: "already_vaccinated",
+          reason_for_refusal_notes: "Already had the vaccine at the GP",
+          patient:
+        )
+      end
+    end
```

Comment:
I think these should be sub-factories as they seem pretty complete versions. I'm also wondering if we need the `after(:create)` part ... I've been playing around and I think these get created correctly with inline association definition.

🥾🥫